extensions/toggl-track/src/toggl/types.ts

```diff
@@ -1,44 +1,29 @@
+export interface Me {
+  default_workspace_id: number;
+}
+
 export interface Workspace {
   id: number;
   name: string;
-  premium: boolean;
-  admin: boolean;
-  default_hourly_rate: number;
-  default_currency: string;
-  only_admins_may_create_projects: boolean;
-  only_admins_see_billable_rates: boolean;
-  rounding: number;
-  rounding_minutes: number;
-  at: Date;
-  logo_url: string;
 }
 
 export interface Project {
-  id: number;
-  wid: number;
-  cid: number;
-  name: string;
   billable: boolean;
-  is_private: boolean;
-  active: boolean;
-  template: boolean;
-  at: Date;
-  created_at: Date;
+  client_id: number;
   color: string;
-  auto_estimates: boolean;
-  actual_hours: number;
-  hex_color: string;
```

Comment:
Did all of these get removed?

Reply:
They still exist, but @rogatty removed unused properties to clean up types.

Reply:
Yup, as I said in the description:
> I decided to remove unused fields instead of adding all the new ones.

I could add _all_ the new fields, but it feels like a waste of time. I can revert and leave it as is, but I wouldn't want to go through all unused fields and verify if they're still (?) correct.

I believe unused fields only add noise, and because they're not verified in action, they can be wrong sometimes. For example `at: Date` I've removed were incorrect, as they should be strings.

Reply:
I'm fine with this as is @rogatty @bkeys818 :)